src/style.css

```diff
@@ -0,0 +1,140 @@
+@import "./stars.css";
+
+.productCard__card {
+  width: 198px;
+  height: 406px;
+
+}
+
+.card {
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.image {
+  background-image: url("./images/imac.jpeg");
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+}
```

Comment:
You can achieve this by a single class. Make it img tag and use it as BEM Element

Reply:
good advice but in checklist they asked to have the external styling for elements and not for blocks ... 
that's the reason I seperated them

Reply:
yesss. I used .image to style everything about the picture
not only this, I concised other parts of css file as well. like button